src/plugins/intel_gpu/src/graph/program.cpp

```diff
@@ -1449,6 +1452,13 @@ std::pair<int64_t, int64_t> program::get_estimated_device_mem_usage() {
     memory_pool pool(get_engine());
     int64_t const_sum = 0;
 
+#ifdef __unix__
```

Comment:
This PR is to address the application kill by OoM killer in linux. Since there is no OoM killer in windows, it will not be killed, thought the swapping takes time.